custom_components/blueprint/translations/en.json

```diff
@@ -1,6 +1,6 @@
 {
+    "title": "Blueprint",
```

Comment:
```suggestion
```

Not needed, it uses `manifest.json` as default

Reply:
Initially I have kept it to show that the user can translate the integration name if needed. In the case of blueprint it's useless but it could in other contexts.
I see different options:

- We keep it like I propose initially.
- We put a comment in the json file to explain the possibility and remove the entry.
- We remove the entry and can be explain in documentation (at least in the cookiecutter documentation)

have you a preference ?

Reply:
A name should never be translated.

Reply:
I think I agree ;-) You convince me I will delete them. Usage too unusual.